### PR TITLE
Fail tile validation if <build><extensions> is present.

### PR DESCRIPTION
**Pull request by peterjanes**
Fixes repaint-io/maven-tiles#94.

### CODE REVIEW

File: src/main/groovy/io/repaint/maven/tiles/TileValidator.groovy [hunk 1]
```diff
@@ -112,6 +112,11 @@ class TileValidator {
 			validModel = null
 		}
 
+		if (model.build?.extensions) {
+			log.error("Tile has extensions and must not have")
+			validModel = null
+		}
+
```
Comment (talios) [inline src/main/groovy/io/repaint/maven/tiles/TileValidator.groovy]: We probably also want to check for `<extensions>true</extensions>` in the general plugin declaration as well ( that's mostly where I've encountered adding extensions personally ).

Comment (peterjanes) [reply src/main/groovy/io/repaint/maven/tiles/TileValidator.groovy]: Good point. Done.

**Comment by peterjanes** [comment]
Hmm. There's nothing in this PR that would affect sitereporting-tiletest, and it passes when run locally.

**Comment by talios** [comment]
@peterjanes I've been seeing that sitereporting IT test fail on the other PRs as well, so _something_ has broken somewhere. Will be jumping on the devlaptop soon and probably merge this, then see if I can spot whats going on there.

**Pull request merged**